K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -167,57 +167,57 @@ private static ExtendedStatsAggregate GetExtendedStatsAggregate(string columnNam
             var jObject = (JObject)row[columnName];
 
             var count = jObject[AggregationsConstants.Count].Value<long?>();
-            extendedStatsAggregate.Count = count ?? (double.IsNaN(count.Value) ? null : count);
+            extendedStatsAggregate.Count = count.HasValue ? double.IsNaN(count.Value) ? null : count : null;
```

Comment:
Why this change? didn't `??` worked fine?

Reply:
Nop; it's raising an error here.
I've made some more tests, and the only way to do it working is to use the `HasValue` property

Reply:
![image](https://user-images.githubusercontent.com/4592555/148962481-2814ea58-0712-4f8f-972f-d16e3235ac9a.png)

Reply:
Ok so the problem is the IsNaN.
So first of all you don't have to check it for the long.

Second of all, I'm not a fan of the double ternarys personally.

What do you think about the alternatives:
1.
```c#
extendedStatsAggregate.Min = (min.HasValue && !double.IsNaN(min.Value)) ? min.Value : null;
```
2.
```c#
var min = jObject[AggregationsConstants.Min]?.Value<double?>() ?? double.NaN;
extendedStatsAggregate.Min = double.IsNaN(min) ? null : min;
```
3.
```c#
extendedStatsAggregate.Min = (min is double d && !double.IsNaN(d)) ? d : null;
```
4.
```c#
extendedStatsAggregate.Min = (min is {} d && !double.IsNaN(d)) ? d : null;
```

Reply:
Your first solution is the best one in my opinion.
Good catch !
As you I was not a big fan of the double ternarys check
Damn, I can't believe I did not found that solution ! :)